leshan-server-core/src/main/java/org/eclipse/leshan/server/bootstrap/BootstrapConfig.java

```diff
@@ -465,9 +480,36 @@ public String toString() {
         }
     }
 
-    @Override
-    public String toString() {
-        return String.format("BootstrapConfig [servers=%s, security=%s, acls=%s, oscore=%s]", servers, security, acls,
-                oscore);
+    public class CipherSuiteId {
+
+        private final byte firstByte;
+        private final byte secondByte;
+
+        public CipherSuiteId(byte firstByte, byte secondByte) {
+            this.firstByte = firstByte;
+            this.secondByte = secondByte;
+        }
+
+        /**
+         * The IANA TLS ciphersuite registry is maintained at
+         * https://www.iana.org/assignments/tls-parameters/tls-parameters.xhtml. As an example, the
+         * TLS_PSK_WITH_AES_128_CCM_8 ciphersuite is represented with the following string "0xC0,0xA8"
+         */
+
+        public CipherSuiteId(ULong valueFromSecurityObject) {
```

Comment:
For this constructor you need to explain(or take the explanation) about how this ULong is created.